Dockerfile

```diff
@@ -39,7 +39,6 @@ EXPOSE 3002
 EXPOSE 3005
 WORKDIR /opt/one-app
 RUN chown node:node /opt/one-app
-USER node
```

Comment:
🤔 I thought we found a workaround to use node as the user instead of root, we need to investigate this change and ensure it doesn't cause any permission issues

Reply:
https://docs.github.com/en/actions/creating-actions/dockerfile-support-for-github-actions#user due to this that wasn't possible. I did try a few different options 

> Docker actions must be run by the default Docker user (root). Do not use the USER instruction in your Dockerfile, because you won't be able to access the GITHUB_WORKSPACE

I tested out the docker images on a module locally and it worked fine. We might need context on why this was added.